packages/e2e/cypress/integration/ui/components/authenticator/sign-up-with-email.feature

```diff
@@ -13,7 +13,7 @@ Feature: Sign Up with Email
     And I don't see "Username" as an input field
     And I don't see "Phone Number" as an input field
 
-@next @react @vue @angular
+@next @react @vue @angular @skip
```

Comment:
This test is leading us to `Exceeded daily email limit for the operation or the account. If a higher limit is required, please configure your user pool to use your own Amazon SES configuration for sending email.`

Reply:
@eddiekeller Didn't you experience this as well & request an increase?

Reply:
@ericclemmons The workaround is to switch to using our own email in Amazon SES rather than relying on Cognito to send out emails for us. Not sure if that is what we want to use long term or not.